src/components/DictionaryCrud/DictionaryCrudData.js

```diff
@@ -23,8 +24,10 @@ function DictionaryCrudData() {
     id: entryId,
     edit: true,
   })
+  const type = data?.type === TYPE_PHRASE ? PHRASES : WORDS
+
   const { onSubmit: createEntry } = useDictionaryEntryCreate()
-  const { onSubmit: updateEntry } = useDictionaryEntryUpdate()
+  const { onSubmit: updateEntry } = useDictionaryEntryUpdate(entryId, type)
```

Comment:
As these props are not being read or used by `useDictionaryUpdate`, best to remove them before merging.

Reply:
oops, missed that one